Traceback (most recent call last):
  File "/Users/filipacalado/opt/anaconda3/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/Users/filipacalado/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/filipacalado/opt/anaconda3/lib/python3.7/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/filipacalado/opt/anaconda3/lib/python3.7/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/filipacalado/opt/anaconda3/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/Users/filipacalado/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 664, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/Users/filipacalado/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/filipacalado/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# for Google Colab ONLY: run this cell

nltk.download('stopwords')
nltk.download('wordnet')
nltk.download('omw-1.4')
nlkt.download('punkt')
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-9-bf3cf756f8c7>[0m in [0;36m<module>[0;34m[0m
[1;32m      4[0m [0mnltk[0m[0;34m.[0m[0mdownload[0m[0;34m([0m[0;34m'wordnet'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0mnltk[0m[0;34m.[0m[0mdownload[0m[0;34m([0m[0;34m'omw-1.4'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 6[0;31m [0mnlkt[0m[0;34m.[0m[0mdownload[0m[0;34m([0m[0;34m'punkt'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;31mNameError[0m: name 'nlkt' is not defined
NameError: name 'nlkt' is not defined

